config/locales/en.yml

```diff
@@ -248,13 +248,22 @@ en:
         ten_years_after_mrc_date: "Enter a tenancy start date that is no more than 10 years after the major repairs completion date"
 
         location:
-          deactivated: "The location %{postcode} was deactivated on %{date} and was not available on the day you entered."
-          reactivating_soon: "The location %{postcode} is not available until %{date}. Select another location or edit the tenancy start date"
-          activating_soon: "The location %{postcode} is not available until %{date}. Enter a tenancy start date after %{date}"
+          deactivated:
+            startdate: "The location %{postcode} was deactivated on %{date} and was not available on the day you entered. Enter a tenancy start date after %{date}"
```

Comment:
Does this make sense? If the location was deactivated wouldn't we want to enter a date `before` it was deactivated instead of `after`?

Reply:
good point, have updated this copy :)